packages/conform-dom/.gitignore

```diff
@@ -1,7 +1,8 @@
 _virtual
+!rollup.config.js
 *.cjs
 *.d.ts
 *.js
 *.mjs
+LICENSE
```

Comment:
We are not copying the LICENSE anymore as tpnpm will handle that for us

Reply:
when I ran `pnpm i` I noticed the prepare script copy it into the folder

Reply:
You might be running an older version. I removed the [settings for copying the LICENSE last night](https://github.com/edmundhung/conform/pull/649/commits/97fe6f6083461e43174e9fe5befa8013865f915e).